src/app/modules/users/upload-confirmation/upload-confirmation.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Inject, Optional, ViewChild } from '@angular/core';
```

Comment:
Please reuse the confirmation modal and remove this if not required

Reply:
We need to change the design in upload confirmation that's why i made separate. previously it is common